### PR TITLE
editoast: add search criteria "ch" for operational points

### PR DESCRIPTION
**Pull request by leovalais**
For PTO import.

### CODE REVIEW

File: editoast/openapi.yaml [hunk 1]
```diff
@@ -3063,6 +3064,16 @@ components:
...
+        track_sections:
+          type: object
+          required:
+            - track
+            - position
+          properties:
+            track:
+              type: string
+            position:
+              type: integer
```
Comment (Castavo) [inline editoast/openapi.yaml]: Shouldn't it be 
```suggestion
        track_sections:
          type: array
          items:
            required:
              - track
              - position
            properties:
              track:
                type: string
              position:
                type: integer
```

**Review by Castavo — APPROVED** [review]
Looks fine to me

**Review by flomonster — APPROVED** [review]
The openapi should be adapted.

File: editoast/openapi.yaml (hunk 1 above)
Comment (leovalais) [reply editoast/openapi.yaml]: Thanks! I'm dying for #3625 to be done...

File: editoast/openapi.yaml [hunk 2]
```diff
@@ -3083,6 +3084,18 @@ components:
...
+          type: array
+          items:
+            type: object
+            required:
+              - track
+              - position
+            properties:
+              track:
+                type: string
+              position:
```
Comment (clarani) [inline editoast/openapi.yaml:3099]: Are you sure it is position and not offset here ?

Comment (leovalais) [reply editoast/openapi.yaml:3099]: Yes it's offset in the pathfinding request but here it's called position 🤷

Comment (leovalais) [reply editoast/openapi.yaml:3099]: I could rename it but that would be done in in pgsql so I'd rather leave it that way 😅

**Pull request merged**